backend/src/middlewares/auth.ts

```diff
@@ -104,7 +104,7 @@ export const verifyDishzeroToken = (req: Request, res: Response, next: NextFunct
  * @returns decoded firebase token in the request object
  */
 export const verifyFirebaseToken = (req: Request, res: Response, next: NextFunction) => {
-    const sessionCookies = req.header('session-token') || req.cookies.session
+    const sessionCookies = req.header('session-token') || req.cookies?.session
     if (!sessionCookies) {
```

Comment:
We may need cookie-parser, I was unable to make this work by setting cookies instead of a header.

Reply:
oh yeah cookie-parser is required.